src/app/platform/keys/list/page.tsx

```diff
@@ -31,10 +31,16 @@ const Page = () => {
   }, []);
 
   const toggleDropdown = (id: string) => {
-    setDropdownStates((prevState) => ({
-      ...prevState,
-      [id]: !prevState[id], 
-    }));
+    setDropdownStates((prevState) => {
```

Comment:
@kunaldevxxx Is there a way we can write a test case for this? A test case will ensure that we don't end up re-creating the bug again.

Reply:
we can do sir but it will take time